src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -341,6 +345,51 @@ export default class PatientViewMutationTable extends React.Component<PatientVie
                 return (MutationAssessorColumnFormatter.filterValue(d).indexOf(filterString) > -1);
             }
         };
+
+        this._columns[MutationTableColumn.COSMIC] = {
+            name: "COSMIC",
+            render: (d:Mutation[])=>CosmicColumnFormatter.renderFunction(d, this.props.cosmicData),
+            sort:(d1:Mutation[], d2:Mutation[], ascending:boolean)=>{
+                return numberSort(CosmicColumnFormatter.getSortValue(d1, this.props.cosmicData),
+                                CosmicColumnFormatter.getSortValue(d2, this.props.cosmicData),
+                                ascending);
+            },
+            tooltip: (<span>COSMIC occurrences</span>)
+        };
+
+        this._columns[MutationTableColumn.TUMOR_ALLELE_FREQ] = {
+            name: "Allele Freq",
+            render: (d:Mutation[])=>AlleleFreqColumnFormatter.renderFunction(d, this.props.sampleManager),
+            sort:(d1:Mutation[], d2:Mutation[], ascending:boolean)=>{
+                return numberListSort(AlleleFreqColumnFormatter.getSortValue(d1, this.props.sampleManager),
+                                AlleleFreqColumnFormatter.getSortValue(d2, this.props.sampleManager),
+                                ascending);
+            },
+            tooltip:(<span>Variant allele frequency in the tumor sample</span>)
+        };
+
+        this._columns[MutationTableColumn.TUMORS] = {
```

Comment:
per above, this column should only be included when there are more than one sample in sampleManager.  see condition above.